### ISSUE TITLE
Inferno test USCDR-13 failed because of a Must Support error for DocumentReference.content.attachment.url in the response.

### ISSUE THREAD

**Issue by IgLysk00** [onc-healthit/inferno-program#261]
**Subject of the issue**
Inferno Single Patient API Document Reference test USCDR-13 is failing because of criteria listed in the Must Support elements; it requires support of content.attachment.data and content.attachment.url (see Inferno output below)
According to HL7 FHIR® US Core Implementation Guide STU3 Release 3.1.1 , section 3.17.1.1 Mandatory and Must Support Data Elements each DocumentReference must have an address (e.g. url) where the document can be retrieved OR the content as inline base64 encoded data.
https://www.hl7.org/fhir/us/core/StructureDefinition-us-core-documentreference.html
![image](https://user-images.githubusercontent.com/73243391/112658697-0a72d300-8e2a-11eb-826c-4e6b8b9c53fc.png)

**Your environment**
* Edition of inferno (Community or Program): Program
* Version of inferno: 1.5.0
* Which browser and version(s) is the bug present on?: Chrome

**Steps to reproduce**
Tell us how to reproduce this issue.
Use a test patient with the following charted Clinical Notes: 
•	Consultation Note (11488-4)
•	Discharge Summary (18842-5)
•	History & Physical Note (34117-2)
•	Procedures Note (28570-0)
•	Progress Note (11506-3)
The JSON output should contain content as inline base64 encoded data and not contain an address (e.g. url) in DocumentReference.content.attachment 

Include:
* The selected test suite: Single Patient API
* The URL of the FHIR server being tested: https://fhir-myrecord-sc.stagingcerner.com/r4/3f2aca24-87f3-4eac-a6d7-1f75247e6b43 (not a public URL)
* The Test ID, if it's a problem with a specific test.: USCDR-13
* Any required configuration options (client ID, client secret, etc.) 
* Anything else needed to make the issue occur.

**Expected behavior**
Tell us what should happen.
Inferno test USCDR-13 should not fail when Document Reference resource contains content as inline base 64 encoded data.

**Actual behavior**
Tell us what happens instead. Include screenshots if possible.

Test Result: Skip
×
 USCDR-13: All must support elements are provided in the DocumentReference resources returned.
US Core Responders SHALL be capable of populating all data elements as part of the query results as specified by the US Core Server Capability Statement. This will look through the DocumentReference resources found previously for the following must support elements:
•	author
•	category
•	content
•	content.attachment
•	content.attachment.contentType
•	content.attachment.data
•	content.attachment.url
•	content.format
•	context
•	context.encounter
•	context.period
•	custodian
•	date
•	identifier
•	status
•	subject
•	type
http://www.hl7.org/fhir/us/core/general-guidance.html#must-support
Message:
Could not find identifier, content.attachment.url in the 17 provided DocumentReference resource(s)
0 Requests

**Comment by yunwwang** [comment]
The "must have" section applies to each instance so that each DocumentReference instance must have either content.attachment.data OR content.attachment.url. This is contained by us-core-6 invariant.
The "must support" section applies to the request responder so that the responder must demonstrate it supports content.attachment.data AND content.attachment.url

**Comment by arscan** [comment]
We are following the spec here, but it comes up again and again, and I agree that it is strange to force responders to support both representations.  I thought this was going to be addressed in v3.2.0 but I'm not seeing any language here http://build.fhir.org/ig/HL7/US-Core/StructureDefinition-us-core-documentreference.html#mandatory-and-must-support-data-elements (I could be just missing it though)

Do you know if changes are expected in v3.2.0 @yunwwang?

**Comment by yunwwang** [comment]
NO. I submitted a JIRA ticket to clarify MUST SUPPORT of DocumentReference.content: https://jira.hl7.org/browse/FHIR-31364

**Comment by IgLysk00** [comment]
Thank you for the prompt response. Upon reviewing your responses and conducting additional testing we can pass test USCDR-13 now. Sorry for taking your time. Please, close this ticket, no longer an issue. Perhaps the error message might be worded differently so it is not misinterpreted by the end user. It makes impression that both elements should be present on every document.

**Comment by arscan** [comment]
I agree that we could improve the message a bit, thanks!
